Web/starter_project_iconics/components/story/PartnersLogos.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+
+const partnerSection = [
```

Comment:
put this on the data section using JSON file, if another partners came along you don't have to touch the code here.